CPAC/pipeline/cpac_pipeline.py

```diff
@@ -947,9 +949,11 @@ def build_T1w_registration_stack(rpool, cfg, pipeline_blocks=None):
     if not rpool.check_rpool('from-T1w_to-template_mode-image_xfm'):
         reg_blocks = [
             [register_ANTs_anat_to_template, register_FSL_anat_to_template],
-             overwrite_transform_anat_to_template,
-             correct_restore_brain_intensity_abcd # ABCD-options pipeline
```

Comment:
We don't need this for `ABCD-options` anymore?

Reply:
They've been moved actually:
https://github.com/FCP-INDI/C-PAC/blob/feature/surface_convenience/CPAC/pipeline/cpac_pipeline.py#L949-L956

Allows for more precise output dir ingressing.